src/command.js

```diff
@@ -43,6 +43,11 @@ export function matchImageSnapshotCommand(defaultOptions) {
           diffOutputPath,
         }) => {
           if (!pass && !added && !updated) {
+            if ((commandOptions.retryCounter || 0) > 0) {
```

Comment:
this can be simplified to 
`if (commandOptions.retryCounter)`

Reply:
done